src/components/Admin/UserPage/ManageUsers.tsx

```diff
@@ -126,15 +225,17 @@ const ManageUsers: () => JSX.Element = () => {
                 <Tr>
                   <Th>{t('users.name')}</Th>
                   <Th>{t('email')}</Th>
-                  <Th>{t('users.isAdmin')}</Th>
+                  <Th>{t('users.adminStatus')}</Th>
                 </Tr>
               </Thead>
               <Tbody>
                 {users.map((user, id) => (
                   <Tr key={id}>
                     <Td>{user.name}</Td>
                     <Td>{user.email}</Td>
-                    <Td>{user.admin ? t('common:yes') : t('common:no')}</Td>
+                    <Td>
```

Comment:
Line 232 (4 lines above this), add a filter for !user.admin so that the admins only appear once on this page. Along wit this, change the text "All Current Users" to something like "Non Admin Users"

Reply:
Done